cosmos/job/drm/drm_ge.py

```diff
@@ -35,8 +37,8 @@ def submit_job(self, task):
 
             task.drm_jobID = unicode(int(out))
         except subprocess.CalledProcessError as cpe:
-            task.log.error('%s submission to %s failed with error %s: %s' %
-                           (task, task.drm, cpe.returncode, cpe.output.decode().strip()))
+            task.log.error('%s submission to %s (%s) failed with error %s: %s' %
+                           (task, task.drm, qsub, cpe.returncode, cpe.output.decode().strip()))
```

Comment:
This change just prints out the offending qsub command when it barfs. Helpful for debugging